app/models/role.rb

```diff
@@ -0,0 +1,2 @@
+class Role < ApplicationRecord
```

Comment:
Add presence validation for name and slug

Reply:
done